src/test/resources/harness-config.yml

```diff
@@ -38,6 +44,12 @@ databasesUnderTest:
     username: C##LIQUIBASE
     password: Secret321
 
+#  - name: oracle
```

Comment:
don't forget to uncomment it before merging. Oracle deserve to be tested ))

Reply:
hahaha! fixed!